app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] KIndly try to add an extra layer of security by asking a user to confirm his or her email in the account creation to make your app more professional and secure. you can try to add `: confirmable` in devise  .you got this :+1:

Reply:
@mugishasam123 Done.